docs/networks/l16-testnet.md

```diff
@@ -13,31 +23,37 @@ sidebar_position: 2
 | Execution Block Explorer URL | [https://ethstats.l16.d.lukso.dev](https://ethstats.l16.d.lukso.dev)                             |
 | Consensus Block Explorer URL | [https://consensus.stats.beta.l16.lukso.network](https://consensus.stats.beta.l16.lukso.network) |
 
+And if you need it, [here is a tutorial on how to do it](https://metamask.zendesk.com/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC).
 
 ## System Requirements
+
 ### Minimum System Requirements
-|       Settings      |      Value      |
-| ------------------- | --------------- |
-| Operating System    | Linux or Darwin |
-| Number of CPU cores | 2               |
-| RAM                 | 16 GB           |
-| SSD                 | 100 GB          |
+
+| Settings            | Value          |
+| ------------------- | -------------- |
+| Operating System    | Linux or MacOS |
+| Number of CPU cores | 2              |
+| RAM                 | 16 GB          |
+| SSD                 | 100 GB         |
 
 ### Recommended System Requirements
-|       Settings      |      Value      |
-| ------------------- | --------------- |
-| Operating System    | Linux or Darwin |
-| Number of CPU cores | 8               |
-| RAM                 | 32 GB           |
-| SSD                 | 512 GB          |
 
-## Running the nodes
+| Settings            | Value          |
+| ------------------- | -------------- |
+| Operating System    | Linux or MacOS |
+| Number of CPU cores | 8              |
+| RAM                 | 32 GB          |
+| SSD                 | 512 GB         |
 
-### Installing dependencies
+Apple's new M1 chips are not supported natively. However, you can follow <br></br> [this guide](https://medium.com/@luki3k5/running-lukso-node-on-m1-mac-acf92d433a38) to run it by using Rosetta, Apple's built-in emulation software.
```

Comment:
```
<br></br>
```

??

2 lines return should do the trick. Or simply `<br>`  (but not `</br>`, this tag does not encapsulate anything)

Reply:
already changed it in other files and might have missed this one. thanks!
FYI: in `.md files`, it is written <br/> instead of </br> as in regular HTML files